apps/code/catalog.universal.i18n

```diff
@@ -208,6 +208,8 @@ PythonTurtleCommandBrown = "'brown'"
 PythonTurtleCommandCircle = "circle(r)"
 PythonTurtleCommandColor = "color('c')/color(r,g,b)"
 PythonTurtleCommandColorWithoutArg = "color(\x11)"
+PythonTurtleCommandColorMode = "colormode(x)"
+PythonTurtleCommandColorModeWithoutArg = "colormode(\x11)"
```

Comment:
No need to define PythonTurtleCommandColorModeWithoutArg!
It should be removed from PythonTurtleCommandColorMode by our algorithm